packages/video_player/video_player/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.2
```

Comment:
Note: if we ended up needing to expose `videoPlayerPlatform`, a minor version bump is required, so 1.1.0.

Reply:
No longer exposing `videoPlayerPlatform`